python/Ganga/Core/GangaRepository/SubJobXMLList.py

```diff
@@ -57,6 +57,8 @@ def __init__(self, jobDirectory='', registry=None, dataFileName='data', load_bac
             return
 
         self._subjobIndexData = {}
+        if parent is not None:
+            self._setParent(parent)
```

Comment:
Edit: I think this check is no longer needed tbh, but I could be wrong. Either way I plan to look into this class in the future for debugging whether subjobs are fully loaded or not.